.github/workflows/update-specs-and-client-libraries.yaml

```diff
@@ -174,6 +174,7 @@ jobs:
           - generator: typescript-axios
             git_repo_id: onfido-node
             preCommit: |
+              sed -iE 's/\([ <{]\)File\([>},]\)/\1FileTransfer\2/g' api.ts
```

Comment:
This manual patch of generated code (to replace `File` with `FileTransfer`) is needed since option below is apparently not supported by the generator:
```
typeMappings:
  File: FileTransfer
```

Reply:
theoretically, it should accept both, right? Otherwise this will be a regression.

Reply:
We've done only pre-releases so far, therefore it should be fine. Adding same method with a different signature for uploading `File` object (without the `FileTransfer` wrapper) wouldn't be easily achievable via the generator unfortunately (that's why also we've went for `FileTransfer` object).

Reply:
it'd still be easier for the customer to just pass a file object (if they have one) rather than creating an intermediate file transfer object; the latter should only be confined to the rare case where the user wants to use in-memory data.

Reply:
I've given that a try, taking advantage from the fact in typescript one can have multiple types of same parameter.
Still to be transformed in template, but the result would be [this](https://github.com/onfido/onfido-node/commit/a6fd28b87592fd33d6083e019d314dfbeb081cf1) one.

The bad things is that this would require user to cast to FileTransfer object in all downloads. We might try to avoid that by differently patching the download methods, but I would avoid that since it's not going to be supported by the officinal (but broken at the moment) `typeMappings` feature. 

I'm torn on what the best approach: old library was already requiring an intermediate wrapper, called `*Request` (e.g. [here](https://github.com/onfido/onfido-node/blob/a66154c972a28afabd19807084a0963951d5050d/src/resources/LivePhotos.ts#L27)), so I believe having to use `FileTransfer` wrapper should not create too many troubles to user.

Reply:
As discussed with @DavidMealha-Onfido, this ambiguity on the return type on endpoints to download files would create more friction and unclarity on users than asking user to wrap file in a `FileTransfer` object.

Unfortunately, we can't have different approach for uploads and downloads methods.